src/syft/ast/globals.py

```diff
@@ -1,4 +1,5 @@
 # from .. import ast # CAUSES Circular import errors
+from typing import Union, List, Optional
```

Comment:
Let's get in the habit of putting these imports on separate lines.

Reply:
I think its time I do the isort fix that will make all of these problems automatically go away! :) 👍